src/network_models/network_constructor.jl

```diff
@@ -6,13 +6,28 @@ function construct_network!(
     buses = PSY.get_components(PSY.Bus, sys)
     bus_count = length(buses)
 
-    get_slack_variables(psi_container.settings) &&
+    get_balance_slack_variables(psi_container.settings) &&
```

Comment:
Minor:  Can we write these as if statements?  Being able to skip the `end` statement doesn't justify this, IMHO.